README.md

```diff
@@ -123,46 +114,24 @@ main(int argc, char** argv) {
 }
 ```
 
-## Developer Guide
-### Dependencies
-```bash
-# for Debian/Ubuntu
-$ ./scripts/deps/install_deps_ubuntu.sh
+## Building from Source
+Please read the [DEVELOPMENT](./DEVELOPMENT.md) guide for instructions on how to build.
+
+
+## How to Contribute
+Contributions are always welcome! See [CONTRIBUTING](./CONTRIBUTING.md) for ways to get started.
 
-# for CentOS/AliOS
-$ ./scripts/deps/install_deps_centos.sh
-```
-### VSAG Build Tool
-```bash
-Usage: make <target>
-
-Targets:
-help:                   ## Show the help.
-debug:                  ## Build vsag with debug options.
-release:                ## Build vsag with release options.
-distribution:           ## Build vsag with distribution options.
-fmt:                    ## Format codes.
-test:                   ## Build and run unit tests.
-asan:                   ## Build with AddressSanitizer option.
-test_asan: asan         ## Run unit tests with AddressSanitizer option.
-tsan:                   ## Build with ThreadSanitizer option.
-test_tsan: tsan         ## Run unit tests with ThreadSanitizer option.
-test_cov:               ## Build and run unit tests with code coverage enabled.
-clean:                  ## Clear build/ directory.
-install:                ## Build and install the release version of vsag.
-```
-### Project Structure
-`benchs/`: benchmark script in Python</br>
-`cmake/`: cmake util functions</br>
-`docker/`: the dockerfile to build develop and ci image</br>
-`examples/`: cpp and python example codes</br>
-`externs/`: third-party libraries</br>
-`include/`: export header files</br>
-`mockimpl/`: the mock implementation that can be used in interface test</br>
-`python_bindings/`: the python bindings</br>
-`scripts/`: useful scripts</br>
-`src/`: the source codes and unit tests</br>
-`tests/`: the functional tests</br>
+
+## Who is using
+VSAG has been integrated with the following database systems.
+
+<img alt="OceanBase" src="" height="60" />
```

Comment:
TBD

Reply:
I'm collecting logos from partners. so maybe you can just remove this TBD section, I will edit it in the next commit